include/zephyr/sys/arch_interface.h

```diff
@@ -1054,7 +1054,13 @@ int arch_gdb_remove_breakpoint(struct gdb_ctx *ctx, uint8_t type,
 #include <zephyr/timing/types.h>
 
 /**
- * @ingroup arch-timing
+ * @brief Arch specific Timing Measurement APIs
+ * @defgroup timing_api_arch Arch specific Timing Measurement APIs
```

Comment:
do you want to add this `doxygengroup::` to Architecture Porting Guide / Timing? Right now it's now showing all the API's you moved under this new group anymore.

Reply:
The timing functions are not strictly required when porting to a new architecture, as long as you don't enable `CONFIG_TIMING_FUNCTIONS`. So I am hesitant in adding to that doc, as it seems to convey the idea that these are required.